game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtils.java

```diff
@@ -302,7 +294,8 @@ private static double findWaterValue(final Territory t, final PlayerId player, f
 
     // Determine value based on nearby territory production
     double nearbyLandValue = 0;
-    final Set<Territory> nearbyTerritories = data.getMap().getNeighbors(t, 3);
+    final Set<Territory> nearbyTerritories =
+        data.getMap().getNeighborsIgnoreEnd(t, 3, ProMatches.territoryCanMoveSeaUnits(player, data, true));
```

Comment:
This change is important because often you have cases where 2 sea territories where within 3 hops but they either aren't connected by sea territories at all or the route over sea territories is very long.